boards/xtensa/m5stickc_plus/m5stickc_plus.dts

```diff
@@ -80,16 +80,65 @@
 	clock-frequency = <I2C_BITRATE_STANDARD>;
 	sda-gpios = <&gpio0 21 GPIO_OPEN_DRAIN>;
 	scl-gpios = <&gpio0 22 GPIO_OPEN_DRAIN>;
-	pinctrl-0 = <&i2c0_sda_gpio21 &i2c0_scl_gpio22>;
+	pinctrl-0 = <&i2c0_default>;
 	pinctrl-names = "default";
+
+	pmic_xpowers_axp192: pmic@34 {
+		compatible = "xpowers,axp192";
+		reg = <0x34>;
+
+		pmic_axp192_regulators: regulators {
+			compatible = "xpowers,axp192-regulator";
+
+			pmic_axp192_buck1: BUCK1 {
+				/* 700mV~3500mV@25mV/step, max output 1200mA */
+				regulator-init-microvolt = <3300000>;
+				regulator-boot-on;
+			};
+
+			pmic_axp192_buck2: BUCK2 {
+				/* 700mV~2275mV@25mV/step, max output 1600mA */
+				regulator-init-microvolt = <2275000>;
+				regulator-boot-on;
+			};
+
+			pmic_axp192_buck3: BUCK3 {
+				/* 700mV~3500mV@25mV/step, max output 700mA */
+				regulator-init-microvolt = <3300000>;
+				regulator-boot-on;
+			};
+
+			pmic_axp192_ldo1: LDO1 {
+				/* Always on */
+				regulator-always-on;
+			};
+
+			pmic_axp192_ldo2: LDO2 {
+				/* 1800mV~3300mV@100mV/step, max output 200mA */
+				regulator-init-microvolt = <3300000>;
+				regulator-boot-on;
+			};
+
+			pmic_axp192_ldo3: LDO3 {
+				/* 1800mV~3300mV@100mV/step, max output 200mA */
+				regulator-init-microvolt = <3300000>;
+				regulator-boot-on;
+			};
+
+			pmic_axp192_ldoio0: LDOIO0 {
```

Comment:
I don't have the schematic of the M5StickC, but are you sure, that this is used as an output?
On M5Stack this pin is used as an GPIO Input (VBUS-Detection)

Reply:
This pin(GPIO0) has different function for different, for StickCPlus it‘s used for MIC power supply.

<img width="734" alt="image" src="https://github.com/zephyrproject-rtos/zephyr/assets/26544331/5bdcff23-9e79-4fc3-941b-43b712efe087">